tests/test_actors.py

```diff
@@ -353,6 +390,51 @@ def test_get_family_by_member_name_returns_expected_data(
     assert response.json() == json.dumps(ACTOR_FAMILY)
 
 
+def test_update_updates_actor(mock_update_actor):
+    client = Client()
+    response = client.actors.v1.update_actor(
+        PARENT_ACTOR_ID, notes="example note", start_date="", end_date=None
+    )
+    assert isinstance(response, Actor)
+    assert response.json() == json.dumps(UPDATED_ACTOR)
+
+
+def test_update_when_parameter_not_provided_does_not_update_parameter(
+    mock_update_actor,
+):
+    client = Client()
+    response = client.actors.v1.update_actor(
```

Comment:
If we can update the defaults on these to be `None` we should update this test to just not pass the named param at all.  I believe `None` is its own data type in python so its not the same as not providing the arg at all 🤔

Reply:
Updated!